docs/02-guides/server-side-events/create-an-event.mdx

```diff
@@ -1,15 +1,158 @@
 ---
 title: Create an event
 description:
-  This guide explains how to create an event in Front-Commerce and dispatch it to external services.
+  In this guide, we will cover the creation of an emitter and how to implement
+  them in your pages, we will create an Emitter that Emit an event when a
+  customer reach a page called FAQ.
 sidebar_position: 1
 ---
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+## What is an event emitter
 
-- Create an event emitter
-- Emit an event from your code
+The events are dispatched through the use of an event emitter. The role of the
+event emitter is to push events to the server event pipeline to be processed by
+the server.
+
+Emitter are implemented in Domain Event class that define an event related to a
+specific event for your app.
+
+## Prerequisites
+
+Before diving into the code, ensure you have the following prerequisites in
+place:
+
+- A Front-Commerce project configured and ready to use
+- [FAQ GraphQL Module](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/tree/main/skeleton/example-extensions/faq-demo/graphql)
+  installed and configured if you want to execute code samples without any
+  changes
```

Comment:
**thought:** I think we should mention it as `FAQ example extension`  and not FAQ graphql module.

**thought:** the FAQ module is not a prerequisite if it basically contains all what the guide explains.

**suggestion:** Rather than listing it in the prerequisites, add in an `:::info` section that the outcome of this guide is already available in the skeleton's FAQ extension.

Reply:
Fixed with your suggestion